recipes/folly/all/conandata.yml

```diff
@@ -38,3 +41,10 @@ patches:
       base_path: "source_subfolder"
     - patch_file: "patches/0015-benchmark-format-macros.patch"
       base_path: "source_subfolder"
+  "2021.05.17.00":
+    - patch_file: "patches/0012-compiler-flags.patch"
+      base_path: "source_subfolder"
+    - patch_file: "patches/0015-benchmark-format-macros.patch"
+      base_path: "source_subfolder"
+    - patch_file: "patches/0016-find-packages-2.patch"
+      base_path: "source_subfolder"
```

Comment:
Seems light 😕

Reply:
"Light"? You mean some patches are missing? I simply removed those that had conflicts with the current sources. I hoped if there was something important, the CI would tell me.

Reply:
Sadly thats not always the case... sometimes it's possible to have things compile but and not produce the correct artifacts or they may not have certain features enabled

Reply:
This one is missing

https://github.com/conan-io/conan-center-index/blob/2236447915d07c7bb6db164013c1263a6ad48d96/recipes/folly/all/patches/0011-disable-logger-example.patch#L9

Looking at the blame it's moved one line too 

https://github.com/facebook/folly/blame/99fbca1df19fdd21f1b831cad6f50ece94573675/folly/CMakeLists.txt#L31

It'll need to be regenerated

Reply:
0009-ill-formed-atomic-copy.patch seems it was fixed https://github.com/facebook/folly/blob/99fbca1df19fdd21f1b831cad6f50ece94573675/folly/fibers/SemaphoreBase.cpp#L170

Reply:
> 0009-ill-formed-atomic-copy.patch seems it was fixed
Then I guess I'm good at this one, right?

> This one is missing
Hmm, I can't see how NOT removing an example might break something.

Reply:
Last time it didnt compile =)

Reply:
Well, it definitely works for me (c). Let's wait for what CI says.